tests/post/steps/test_monitoring.py

```diff
@@ -58,7 +58,7 @@ def _wait_job_status():
 
     utils.retry(
         _wait_job_status,
-        times=20,
+        times=30,
```

Comment:
Could also be good to explain why we increase the timing here, what/how it's suppose to solve something.

Reply:
done